### DISCUSSION TITLE
Notch

### DISCUSSION THREAD

**Discussion by kamilm8** [bttv-android/bttv#410] [Ideas]
Hello! Ur app very good Okayeg 
But I have black bar on the left because of my single camera, can make an option for this? Extra Twitch had it :)

**Comment by FoseFx** [comment]
I think this is [out of scope](https://github.com/bttv-android/bttv/blob/master/CONTRIBUTING.md#scope-of-the-project). You should definitely try to ask [Twitch about it](https://help.twitch.tv/s/contactsupport) though.
